### DISCUSSION TITLE
What's next for Totems+?

### DISCUSSION THREAD

**Discussion by ben-wals** [The-Iceburg/TotemsPlus#28] [General]
We're extremely proud of how far Totems+ has come since its creation in November 2021 to our first fully featured release (#25) but despite all the excitement the question now stands: What's next for Totems+?

Sure, for version #25 we've implemented a tonne of features but we still have loads more ideas for Totems+. So we thought we should make this post outlining mainly for ourselves what we wish to achieve with Totems+ before our next major release V2.0.0

-  Bugfixes
The features we want to add and the features we've added are still full of bugs and we want to do our best at prioritising their fixing during the creation of V2. However this is relatively hard as there is only 2 of us to test so if you spot any issues create an issue on this GitHub page with the tag bug or tweet us @theiceburg21

- Refactoring
We understand our code (the majority of it) was written at a time were our knowledge was limited so for V2 we want to commit to refactoring perhaps not all of our code but at least some of it for a much smoother and cleaner experience especially with all the new features we want to add.

- New Features
Here's the best part, the new features we already have a list of ideas we want to add that we made during work on V1 and I'm sure it will only grow as we develop V2 but at the time of writing we are looking to implement:

1.   #7
2. #9
3. #10
4. #11
5. #12
6. #13

- GitHub Skills
We also want to improve our skills with using this amazing platform to improve our workflow and make the distribution and understanding of our code even better so by V2 you should have seen a lot of changes around here :)

**Comment by Dockuin** [comment]
Very Cool! Looking forward to working on V2.0.0

**Reply by ben-wals** [reply]
It's gunna be fun! 🙄
